set.mm

```diff
@@ -638670,6 +638670,162 @@ deduction version (Contributed by metakunt, 22-May-2024.) $)
       WHAVKVLVNCAGDVEZABWEWFVBWGVFVGVHAVSWNABVRWEWFAVKVLCWOWGVIVJVGTTT $.
   $}
 
+  ${
+    $d A x $.  $d A y $.  $d B x $.  $d B y $.  $d ph x $.  $d ph y $.
+    dvrelog2.1 $e |- ( ph -> A e. RR ) $.
+    dvrelog2.2 $e |- ( ph -> B e. RR ) $.
+    dvrelog2.3 $e |- ( ph -> 0 < A ) $.
+    dvrelog2.4 $e |- ( ph -> A <_ B ) $.
+    dvrelog2.5 $e |- F = ( x e. ( A [,] B ) |-> ( log ` x ) ) $.
+    dvrelog2.6 $e |- G = ( x e. ( A (,) B ) |-> ( 1 / x ) ) $.
+    $( The derivative of the logarithm, ~ ftc2 version.  (Contributed by
+       metakunt, 11-Aug-2024.) $)
+    dvrelog2 $p |- ( ph -> ( RR _D F ) = G ) $=
+      ( cr cdv co clog crp cc wcel cc0 vy cioo c1 cv cdiv cmpt cicc wceq oveq2d
+      cfv a1i crn ctg ccnfld ctopn cpr reelprrecn rpssre ax-resscn sstri adantl
+      wa sseli wne rpne0 logcld 1red redivcld cres csn cdif wf wf1o logf1o f1of
+      ax-mp wss cin c0 wn disjsn mpbir disjdif2 ssdif eqsstrri feqresmpt eqcomd
+      0nrp dvrelog eqtrd clt wbr cle w3a wb elicc2 syl2anc biimpa simp1d adantr
+      0red simp2d ltletrd jca elrp sylibr ex ssrdv eqid tgioo2 iccntr dvmptres2
+      cnt ) AMENOZBCDUBOZUCBUDZUEOZUFZFAXNMBCDUGOZXPPUJZUFZNOXRAEYAMNEYAUHAKUKU
+      IABXTXQMUBULUMUJZUNUOUJZMQXOXSMMRUPSAUQUKAXPQSZVBXPYDXPRSAQRXPQMRURUSUTZV
+      CVAYDXPTVDAXPVEZVAVFYDXQMSAYDUCXPYDVGQMXPURVCYFVHVAAMBQXTUFZNOMPQVIZNOZBQ
+      XQUFZAYGYHMNAYHYGABRTVJZVKZPULZQPYLYMPVLZAYLYMPVMYNVNYLYMPVOVPUKQYLVQAQQY
+      KVKZYLQYKVRVSUHZYOQUHYPTQSVTWHQTWAWBQYKWCVPQRVQYOYLVQYEQRYKWDVPWEUKWFWGUI
+      YIYJUHABWIUKWJAUAXSQAUAUDZXSSZYQQSZAYRVBZYQMSZTYQWKWLZVBYSYTUUAUUBYTUUACY
+      QWMWLZYQDWMWLZAYRUUAUUCUUDWNZACMSZDMSZYRUUEWOGHCDYQWPWQWRZWSZYTTCYQYTXAAU
+      UFYRGWTUUIATCWKWLYRIWTYTUUAUUCUUDUUHXBXCXDYQXEXFXGXHYCYCXIZXJUUJAUUFUUGXS
+      YBXMUJUJXOUHGHCDXKWQXLWJAFXRFXRUHALUKWGWJ $.
+  $}
+
+  ${
+    $d A x $.  $d A y $.  $d B x $.  $d B y $.  $d ph x $.  $d ph y $.
+    dvrelog3.1 $e |- ( ph -> A e. RR ) $.
+    dvrelog3.2 $e |- ( ph -> B e. RR ) $.
+    dvrelog3.3 $e |- ( ph -> 0 < A ) $.
```

Comment:
Then @benjub's remark should be applied here:  0 < A can be weakened to 0 ≤ A

Reply:
Fixed it, although I won't really need the full strength of the statement.